intro-to-R-tidyverse/03-intro_to_r_tidyverse_exercise.Rmd

```diff
@@ -0,0 +1,186 @@
+---
+title: 'Exercise: Clean Metadata and Prepare Gene Expression Matrix for Downstream
+  Analysis'
+output:
+  html_notebook:
+    toc: yes
+    toc_float: yes
+---
+The goal of these exercises is to help you get familiar with using tidyverse 
+functions to manipulate data. 
+
+In this example, we will use a [medulloblastoma dataset](https://www.refine.bio/experiments/GSE49243/gene-expression-data-from-medulloblastoma-tumor-samples) 
+that has been processed on [refine.bio](http://docs.refine.bio/en/latest/main_text.html#processing-information). 
+In short, this study collected medulloblastoma microarray profiles from patients
+of a variety of ages and includes information about their demographics and 
+histology of the medulloblastoma samples [(Kool et al. _Cancer Cell_ . 2014.)](https://www.ncbi.nlm.nih.gov/pubmed/24651015).
+
+### Setup 
+Before we start the exercises, we need to create the necessary directories and 
```

Comment:
I find it easier to read the Rmd file in RStudio if there is a space between headers and the text beneath. For instance, this would become
```
### Setup

Before we start the exercises...
```
I would carry this change throughout.